drivers/counter/counter_dtmr_cmsdk_apb.c

```diff
@@ -63,9 +63,28 @@ static u32_t counter_dtmr_cmsdk_apb_read(struct device *dev)
 	return value;
 }
 
+static int counter_dtmr_cmsdk_apb_set_wrap_alarm(struct device *dev,
+						 u32_t ticks,
+						 counter_wrap_callback_t callback,
+						 void *user_data)
+{
+	return -ENODEV;
+}
+
+static u32_t counter_dtmr_cmsdk_apb_get_wrap(struct device *dev)
+{
+	return DUALTIMER_MAX_RELOAD;
+}
+
 static int counter_dtmr_cmsdk_apb_set_alarm(struct device *dev,
```

Comment:
if `counter_config_info` is correctly set you don't need to provide this function as it will be never called if counter_config_info indicates that there is not channels. See `counter_set_channel_alarm` in counter.h